lib/oidcstrategy.js

```diff
@@ -198,8 +196,7 @@ function Strategy(options, verify) {
   this._verify = verify;
   this._passReqToCallback = !!options.passReqToCallback;
 
-  this._stateHandler = new SessionContentHandler('state', options.stateMaxAmount, options.stateMaxAge);
-  this._nonceHandler = new SessionContentHandler('nonce', options.nonceMaxAmount, options.nonceMaxAge);
+  this._sessionContentHandler = new SessionContentHandler(options.sessionContentMaxAmount || 10, options.sessionContentMaxAge || 3600);
```

Comment:
The 'options' is read from the config file. If user doesn't provide values in config, we set the default value here. You mean user must provide the value in config, if they don't provide a value, we throw an error?